modules/javafx.media/src/main/native/gstreamer/3rd_party/libffi/src/prep_cif.c

```diff
@@ -1,7 +1,7 @@
 /* -----------------------------------------------------------------------
    prep_cif.c - Copyright (c) 2011, 2012, 2021  Anthony Green
                 Copyright (c) 1996, 1998, 2007  Red Hat, Inc.
-                Copyright (c) 2022 Oracle and/or its affiliates.
+                Copyright (c) 2022, 2024, Oracle and/or its affiliates.
```

Comment:
Revert this change. The copyright years should not be modified in a third-party file.